cmd/preview/list_test.go

```diff
@@ -0,0 +1,103 @@
+package preview
+
+import (
+	"context"
+	"fmt"
+	"github.com/okteto/okteto/internal/test/client"
+	"github.com/okteto/okteto/pkg/okteto"
+	"github.com/okteto/okteto/pkg/types"
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func Test_listPreview(t *testing.T) {
+	ctx := context.Background()
+	opts := ListFlags{}
+	var tests = []struct {
+		name     string
+		previews []types.Preview
+		err      error
+	}{
+		{
+			name: "List all previews",
+			previews: []types.Preview{
+				{
+					ID: "test",
+				},
+				{
+					ID: "test-1",
+				},
+			},
+		},
+		{
+			name:     "error retrieving preview list",
+			previews: nil,
+			err:      fmt.Errorf("error retrieving previews"),
+		},
```

Comment:
I think we are missing a test case that checks the flow when the preview list returns a user error
``````go

{
	name:     "user error retrieving preview list",
	previews: nil,
	err:      oktetoErrors.UserError{},
}
``````

Reply:
Resolved